### ISSUE TITLE
Need Real-Time collaboration System

### ISSUE THREAD

**Issue by AtomGrafiks** [adobe/brackets#11177]
Please need real-time collaboration system as Google Drive !

**Comment by fcardona** [comment]
+1. You read my mind.

**Comment by ApfelUser** [comment]
+1 that would be a killer-feature!

**Comment by adelriosantiago** [comment]
+1!!! Has anyone tested this? https://github.com/simphax/brackets-rtcollab

**Comment by dpeukert** [comment]
:+1:

**Comment by Nosthertus** [comment]
:+1:

**Comment by jackycute** [comment]
Hey everyone, I made a realtime collaborative markdown note on all platforms.
It called HackMD https://hackmd.io, please try it out, it's open source and free now.

I use Brackets for almost a year and I love it every much.
Actually some ideas of HackMD it stolen form Brackets.
We are planning to do a really big revision and gathering new ideas.
And I'm thinking if that's cool to contribute a plugin that integrate Brackets with HackMD.
Or just make collaborate possible over the same architecture?

If you have any ideas please let me know.

**Comment by sakshamgoyal01** [comment]
HOW TO USE??????????

**Comment by jackycute** [comment]
@sakshamgoyal01
You mean HackMD?
I haven't make it as extension yet.
But you could try online version - https://hackmd.io

**Comment by sakshamgoyal01** [comment]
NO.... how to use in brackets?? i installed it but i dont know what to do next

**Comment by RationalCoding** [comment]
Currently just a working prototype, but I've created the [multihack-brackets](https://github.com/RationalCoding/multihack-brackets) extension to address this. 

Any feedback is appreciated!
